### PR TITLE
Pu/lookupfix

### LINKED ISSUE #787: Vlookup approximate lookup not working
### Description

When vloopup last argument is true it still searches for exact results while in excel it searched for approximate results

### Steps to reproduce

1. add a lookup table with first column all numbers in sorted order
2. add vlookup and search for a number inbetween
3. no results
check [demo](https://codesandbox.io/s/competent-jackson-490o7?file=/src/data.js)

### Demo
https://codesandbox.io/s/competent-jackson-490o7?file=/src/hyperformulaConfig.js

### Your environment
node10.15.3/chrome 92.0.4515.107(64bit)

* HyperFormula version: 1.0.0
* Browser Name and version: chrome 92.0.4515.107(64bit)
* Operating System :windows 10

### PR DESCRIPTION
**Pull request by izulin**
### Context

Fixes #787 

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1.
2.
3.

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation,
- [x] I described the modification in the CHANGELOG.md file.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+- `binarySearchThreshold` option is now deprecated, as every search of sorted data always uses binary search.
```
Comment (wojciechczerniak) [inline CHANGELOG.md:10]: Can you add `deprecated` tag to the `binarySearchThreshold` option description?
ie `* @deprecated Every search of sorted data always uses binary search`

**Review by wojciechczerniak — APPROVED** [review]
👍

File: CHANGELOG.md (hunk 1 above)
Comment (izulin) [reply CHANGELOG.md:10]: in the Config?

Comment (wojciechczerniak) [reply CHANGELOG.md:10]: yes, it will be displayed in the API reference

**Pull request merged**